app/src/main/java/io/gnosis/safe/ui/transactions/details/viewdata/TransactionDetailsViewData.kt

```diff
@@ -0,0 +1,196 @@
+package io.gnosis.safe.ui.transactions.details.viewdata
```

Comment:
I don't like this code duplication. We should discuss whether we remove it and use only TransactionDetails model. We could pass model and map of SolidityAddress to AddressInfoData to updateUi instead of TransactionDetailsViewData and avoid code duplication. Our TransactionDetailsViewData could be simplified to 
```data class TransactionDetailsViewData(val txDetails: TransactionDetails, knownAddressData: Map<Solidity.Address, AddressInfoData>)```

Reply:
Can we do this in a future PR?

Reply:
yes, could you open an issue for this please